testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/xsite/XSiteBackupForTestCase.java

```diff
@@ -1,189 +0,0 @@
-/*
- * JBoss, Home of Professional Open Source.
- * Copyright 2011, Red Hat, Inc., and individual contributors
- * as indicated by the @author tags. See the copyright.txt file in the
- * distribution for a full listing of individual contributors.
- *
- * This is free software; you can redistribute it and/or modify it
- * under the terms of the GNU Lesser General Public License as
- * published by the Free Software Foundation; either version 2.1 of
- * the License, or (at your option) any later version.
- *
- * This software is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this software; if not, write to the Free
- * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
- * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
- */
-package org.jboss.as.test.clustering.xsite;
-
-import java.io.IOException;
-import java.net.URI;
-import java.net.URISyntaxException;
-import java.net.URL;
-
-import javax.servlet.http.HttpServletResponse;
-
-import org.apache.http.HttpResponse;
-import org.apache.http.client.methods.HttpGet;
-import org.apache.http.client.utils.HttpClientUtils;
-import org.apache.http.impl.client.CloseableHttpClient;
-import org.jboss.arquillian.container.test.api.Deployment;
-import org.jboss.arquillian.container.test.api.OperateOnDeployment;
-import org.jboss.arquillian.container.test.api.RunAsClient;
-import org.jboss.arquillian.container.test.api.TargetsContainer;
-import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.arquillian.test.api.ArquillianResource;
-import org.jboss.as.test.clustering.cluster.ExtendedClusterAbstractTestCase;
-import org.jboss.as.test.http.util.TestHttpClientUtils;
-import org.jboss.shrinkwrap.api.Archive;
-import org.jboss.shrinkwrap.api.ShrinkWrap;
-import org.jboss.shrinkwrap.api.asset.StringAsset;
-import org.jboss.shrinkwrap.api.spec.WebArchive;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-/**
- * Test xsite functionality on a 4-node, 3-site test deployment:
- *
- * sites:
- *   LON: {LON-0, LON-1}  // maps to CONTAINER_1/CONTAINER_2 (see arquillian.xml)
- *   NYC: {NYC-0}         // maps to CONTAINER_3 (see arquillian.xml)
- *   SFO: {SFO-0}         // maps to CONTAINER_4 (see arquillian.xml)
- *
- * routes:
- *   LON -> NYC,SFO
- *   NYC -> LON
- *   SFO -> LON
- *
- * backups: (<site>:<container>:<cache>)
- *   LON:web:repl backed up by NYC:web:repl, SFO:web:LONrepl
- *   NYC not backed up
- *   SFO not backed up
- *
- *
- * @author Richard Achmatowicz
- */
-@RunWith(Arquillian.class)
-@RunAsClient
-public class XSiteBackupForTestCase extends ExtendedClusterAbstractTestCase {
```

Comment:
FYI I am already removing the test in https://github.com/wildfly/wildfly/pull/10697

Reply:
Then it's a race to see which gets merged first... :)